jenkins-scripts/docker/reprepro_updater.bash

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -x
+
+set -e
+
+# Knowing Script dir beware of symlink
+[[ -L "${0}" ]] && SCRIPT_DIR=$(readlink "${0}") || SCRIPT_DIR="${0}"
+SCRIPT_DIR="${SCRIPT_DIR%/*}"
+
+PYTHON_VENV="${WORKSPACE}/venv"
+REPREPRO_GIT_REPO="https://github.com/j-rivero/reprepro-updater"
```

Comment:
Should the bug be fixed upstream so the fork isn't needed? I don't see a PR associated with this issue?

Reply:
Problem is that I think changes are not strictly bugs but changes on behavior. With the option of only accepting critical fixes in that repo I don't think that my changes fit well into that category.

Reply:
@nuclearsandwich and I agree on moving my fork to reprepro-updater.

Reply:
a83445c949e6e2feb1ab4af797e4c127b6892333